lib/model/model.go

```diff
@@ -675,19 +675,20 @@ func (m *Model) Index(deviceID protocol.DeviceID, folder string, fs []protocol.F
 
 	m.fmut.RLock()
 	files, ok := m.folderFiles[folder]
-	runner := m.folderRunners[folder]
```

Comment:
I wouldn't make this change. Consider that each RLock/RUnlock cycle is fairly expensive, while the map lookups and the `if !ok` are essentially free.